### PR TITLE
Edit Session Info (all fields)

### PR DESCRIPTION
**Pull request by wisemandenny**
This PR addresses #376 - `Sessions` should be editable.

You can now edit all of the session fields as an admin.

### CODE REVIEW

File: frontend/src/views/sessions/index.tsx [hunk 1]
```diff
@@ -68,7 +64,7 @@ export function AdminSessionsView() {
                         setAddDialogVisible(false);
                     }}
                 />
-                <ConnectedSessionList />
+                <SessionsList />
```
Comment (siefkenj) [inline frontend/src/views/sessions/index.tsx]: Rename it `ConnectedSessionList` to be consistent with the rest of the code.

File: frontend/src/views/sessions/index.tsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from "react";
-import { connect, useSelector } from "react-redux";
+import { useSelector } from "react-redux";
 import { ConnectedAddSessionDialog } from "./add-session-dialog";
-import { sessionsSelector, activeSessionSelector } from "../../api/actions";
-import { SessionsList } from "../../components/sessions";
+import { activeSessionSelector } from "../../api/actions";
+import { SessionsList as ConnectedSessionsList } from "../../components/sessions";
```
Comment (siefkenj) [inline frontend/src/views/sessions/index.tsx]: No....the convention in this codebase is that every widget connected to the Redux store has a name that starts with `Connected`. Changing it on import confuses the matter.

Comment (wisemandenny) [reply frontend/src/views/sessions/index.tsx]: I made a big brain fart here. Let me fix that...

**Comment by wisemandenny** [comment]
i broke it again. going to reopen soon...

**Pull request closed without merging**